homeassistant/components/oncue/entity.py

```diff
@@ -52,4 +52,6 @@ def _oncue_value(self) -> str:
         """Return the sensor value."""
         device: OncueDevice = self.coordinator.data[self._device_id]
         sensor: OncueSensor = device.sensors[self.entity_description.key]
+        if sensor.value == VALUE_UNAVAILABLE:
+            return STATE_UNAVAILABLE
```

Comment:
I like it.  Changed it to use that property.  Also I fixed an issue with the availability of the battery voltage.  In my testing, the cloud reports a 0 for this when the generator is disconnected rather than "--" like all the other sensors.  Usually battery voltage is 13.4 and a zero value would mean a dead battery....  Anyways, added additional logic into the property to make data also unavailable if the cloud is saying the generator is disconnected.